package.json

```diff
@@ -7,15 +7,19 @@
     "type": "git",
     "url": "git://github.com/medic/medic-couch2pg.git"
   },
+  "bin": {
+    "replicate-medic": "./index.js"
```

Comment:
@henokgetachew I also added this so you can boot it the same way as 3.x

Reply:
Yeah this is pretty cool.